CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 1.8.6 (Upcoming)
 
+### NOTES:
+* hcp: users of HCP for storing metadata for their builds will see some changes in
```

Comment:
I think just saying "HCP Packer" implies that they store metadata for the builds.
```suggestion
* hcp: users of HCP Packer will see some changes in
```

Reply:
:+1: